presto-jmx/src/main/java/com/facebook/presto/connector/jmx/JmxStatsHolder.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.connector.jmx;
+
+import com.google.common.collect.EvictingQueue;
+import com.google.common.collect.ImmutableList;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+public class JmxStatsHolder
+{
+    private final List<String> tables;
+    private final Map<String, EvictingQueue<List<Object>>> tablesRows = new HashMap<>();
+
+    public JmxStatsHolder(int evictionLimit, List<String> tableNames)
+    {
+        tables = ImmutableList.copyOf(tableNames);
```

Comment:
simply no. There are too many methods called "copyOf". Especially that I have just changed that field to `Set<String>`